UsanaseD/TEAMWORK
# #168858820 bug(Ui-correction) corrects my UI layout

#### What does this PR do?
It corrects My UI Layout
#### Description of Task to be completed?
MY UI dashboard layout was changed and it now looks clean with few elements I also fixed my divs allignments
#### How should this be manually tested?
head to https://usanased.github.io/TEAMWORK/UI/ click sign in you will see that it has changed
#### Any background context you want to provide?
sorry I don't get the question
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/168858820
#### Screenshots (if appropriate)
![Capture](https://user-images.githubusercontent.com/47520560/65931204-f868cd80-e408-11e9-8ae0-724cb73b5c14.PNG)


#### Questions: from the example you gave us on commit message you ended with the word "ends followed by story id" yet i use starts,where is the difference?
